src/formatter.rs

```diff
@@ -102,29 +110,34 @@ impl Settings {
                 Rule::compound_atomic_modifier => modifier = pair.as_str().to_string(),
                 Rule::expression => {
                     let s = self.format_expression(pair);
-                    if start == end {
-                        code = format!("{{{}}}", s.join("|"));
-                    }
-                    else if self.choice_first {
-                        let space = std::iter::repeat(' ').take(self.indent - 2).collect::<String>();
-                        code = format!("{{\n  {}}}", indent(&s.join("\n| "), &space));
-                    }
-                    else {
-                        let space = std::iter::repeat(' ').take(self.indent).collect::<String>();
-                        code = format!("{{\n{}}}", indent(&s.join(" |\n"), &space));
+                    match s {
+                        Ok(s) => {
+                            if start == end {
```

Comment:
Oh please..

Reply:
Hello @ice1000,

I dont understand your comment and would like to learn from this interaction. Could you elaborate on "Oh please.."

Thanks for you time in this review.
Dictated but not reviewed,
Becker

Reply:
It's also about using `?` instead of pattern matching

Reply:
Hello @ice1000 ,

That is a good suggestion. Had you been clear about your comment the first time I might have committed it.

Thanks
Dictated but not read.
Becker